libsplinter/src/events/ws.rs

```diff
@@ -338,6 +414,9 @@ impl<T: ParseBytes<T> + 'static> WebSocketClient<T> {
                                                 err
                                             );
                                         }
+                                        if let Err(err) = context.try_reconnect() {
```

Comment:
Should it try in this case?  Isn't this an explicit close?

Reply:
Yes, this is called if the ws gets a close message from the server. I talked with @agunde406 about this, I guess the expectation was that even if the splinter daemon gracefully closes the connection the gameroomd should try to re-establish with the expectation that the server will return soon? The gameroom does work properly without those connections.